cli/root.go

```diff
@@ -101,7 +101,7 @@ func Core() []*cobra.Command {
 func AGPL() []*cobra.Command {
 	df := deployment.Flags()
 	all := append(Core(), Server(df, func(_ context.Context, o *coderd.Options) (*coderd.API, error) {
-		o.DeploymentFlags = &df
+		o.DeploymentFlags = df
```

Comment:
Why not just set these in the `Server` func instead? That way it doesn't have to be set for Enterprise and AGPL

Reply:
I _think_ I need to do it like this so we don't make and set 2 different structs - because the flags use the pointer reference this is important. But if I'm misunderstanding something please correct me. @deansheather was very helpful when _I_ was writing this.

Reply:
This line specifically isn't required and Garrett removed it, however we need to pass in the dflags instance instead of creating it in the server handler because we need to attach enterprise flags here as well